src/components/Navigation/Navigation.tsx

```diff
@@ -41,12 +47,30 @@ function Nav({ isAuthenticated = false, className = "" }: NavProps) {
         }}
       />
 
+      <Link
+        className="nav-link nav-basket"
+        text={cart ? `(${calculateTotalQuantity()})` : "(0)"}
+        icon={<FaShoppingCart />}
+        href="/basket"
+        onClick={(e) => {
+          e.preventDefault();
+          navigate("/basket");
```

Comment:
they are doing the same... just update Link component to render `<a/>` for external links and react-router's `Link` for internal ones